<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_about</class>
 <widget class="QDialog" name="dlg_about">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>349</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>412</width>
    <height>349</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>412</width>
    <height>349</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>关于 Xware Desktop</string>
  </property>
  <property name="windowIcon">
   <iconset theme="xware-desktop"/>
  </property>
  <property name="locale">
   <locale language="Chinese" country="China"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>299</y>
     <width>392</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>项目主页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_homepageLink">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&lt;a href=&quot;https://github.com/Xinkai/XwareDesktop/wiki&quot;&gt;https://github.com/Xinkai/XwareDesktop/wiki&lt;/a&gt;</string>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>获取代码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_repoLink">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&lt;a href=&quot;https://github.com/Xinkai/XwareDesktop&quot;&gt;https://github.com/Xinkai/XwareDesktop&lt;/a&gt;</string>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>391</width>
     <height>231</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_brief">
    <attribute name="title">
     <string>简介</string>
    </attribute>
    <widget class="QLabel" name="label_info">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>371</width>
       <height>201</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Xware Desktop作为Xware（迅雷固件）的前端，是一次将用于嵌入式设备的迅雷软件带到Linux桌面的一次尝试。本软件与迅雷官方没有任何关系。 &lt;/p&gt;&lt;p&gt;Xware Desktop的目标是提供与Windows平台上迅雷相当的功能。你可以这样认为： &lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#005500;&quot;&gt;Xware Desktop是&lt;/span&gt;&lt;span style=&quot; font-weight:600; color:#005500;&quot;&gt;迅雷在Linux上的原生开源方案&lt;/span&gt;&lt;span style=&quot; color:#005500;&quot;&gt;，无需Wine。&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#005500;&quot;&gt;Xware Desktop是&lt;/span&gt;&lt;span style=&quot; font-weight:600; color:#005500;&quot;&gt;半官方半社区的迅雷&lt;/span&gt;&lt;span style=&quot; color:#005500;&quot;&gt;：下载核心使用迅雷原版xware，界面使用开源的代码。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_versions">
    <attribute name="title">
     <string>版本详情</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>371</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Xware Desktop</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_githash">
        <property name="text">
         <string>开发版</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Xware</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_xwareVer">
        <property name="text">
         <string>xware</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Python</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_pythonVer">
        <property name="text">
         <string>python</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Qt</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_qtVer">
        <property name="text">
         <string>qt</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Qt.WebKit</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_qtwebkitVer">
        <property name="text">
         <string>webkit</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Python.PyQt</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_pyqtVer">
        <property name="text">
         <string>pyqt</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Python.pyinotify</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_pyinotifyVer">
        <property name="text">
         <string>pyinotify</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Python.aiohttp</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_aiohttpVer">
        <property name="text">
         <string>aiohttp</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>13</y>
     <width>222</width>
     <height>39</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Xware Desktop</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_version">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="rc/resource.qrc"/>
 </resources>
 <connections/>
</ui>
